source/1-SDLC-organization/package.json

```diff
@@ -29,7 +29,7 @@
     "@aws-cdk/aws-codepipeline-actions": "1.85.0",
     "@aws-cdk/core": "1.85.0",
     "@aws-cdk/pipelines": "1.85.0",
-    "aws-bootstrap-kit": "0.2.9",
+    "aws-bootstrap-kit": "file:../../../aws-bootstrap-kit/source/aws-bootstrap-kit/dist/js/aws-bootstrap-kit@0.3.0.jsii.tgz",
```

Comment:
Will be changed to 0.3.0 when https://github.com/awslabs/aws-bootstrap-kit/pull/17 is merged